### PR TITLE
added "show average" setting

### PR DESCRIPTION
**Pull request by OctahedronV2**
### Description

I did the thing, I added the setting!!!

### CODE REVIEW

File: frontend/src/scripts/db.ts [hunk 1]
```diff
@@ -313,11 +313,11 @@ export async function getUserAverageWpm10<M extends MonkeyTypes.Mode>(
 
     // Return the last 10 average wpm for quote if the current quote id has never been completed before by the user.
     if (count == 0 && mode == "quote") {
-      return Math.round(last10Wpm / last10Count);
+      return last10Wpm / last10Count;
     }
 
     // Return the average wpm of the last 10 completions for the targeted test mode.
-    return Math.round(wpmSum / count);
+    return wpmSum / count;
```
Comment (Miodec) [inline frontend/src/scripts/db.ts:320]: If youre going to return unrounded data, you need to check where this funciton is used and round it there. Im pretty sure this breaks pace caret.

File: frontend/src/scripts/elements/commandline-lists.ts [hunk 2]
```diff
@@ -358,6 +358,29 @@ const commandsLiveWpm: MonkeyTypes.CommandsGroup = {
   ],
 };
 
+const commandsShowAvg: MonkeyTypes.CommandsGroup = {
+  title: "Show Average...",
```
Comment (Miodec) [inline frontend/src/scripts/elements/commandline-lists.ts:362]: Lowercase a

**Review by Miodec — CHANGES_REQUESTED** [review]
Also, what happens if you try to use this while signed out?

File: frontend/src/scripts/db.ts (hunk 1 above)
Comment (OctahedronV2) [reply frontend/src/scripts/db.ts:320]: It doesn't break pace caret, I checced. I'm pretty sure if anything it just makes it more accurate lol. :))))))))

File: frontend/src/scripts/elements/commandline-lists.ts (hunk 2 above)
Comment (OctahedronV2) [reply frontend/src/scripts/elements/commandline-lists.ts:362]: ughhhhhhHHHHHhHHH

**Review by OctahedronV2 — COMMENTED** [review]
love u mio

File: frontend/src/scripts/db.ts (hunk 1 above)
Comment (Miodec) [reply frontend/src/scripts/db.ts:320]: Please round

**Review by Miodec — DISMISSED** [review]
Thank you mister octa

**Pull request merged**